packages/app/package.json

```diff
@@ -25,6 +25,7 @@
     "eslint": "^8.11.0",
     "eslint-config-next": "canary",
     "postcss": "^8.4.23",
+    "sharp": "^0.32.1",
```

Comment:
Why are we adding this lib?

Reply:
I don't think we need this.

Reply:
https://nextjs.org/docs/messages/install-sharp